cloud/services/groups/groups.go

```diff
@@ -76,11 +76,11 @@ func (s *Service) Delete(ctx context.Context) error {
 		return errors.Wrapf(err, "failed to delete resource group %s", s.Scope.ResourceGroup())
 	}
 
-	s.Scope.V(2).Info("successfully deleted resource group", "resource group", s.Scope.ResourceGroup())
+	s.Scope.V(2).Info("Deleting resource group. It might take a while for all resources to be completely gone...", "resource group", s.Scope.ResourceGroup())
 	return nil
 }
 
-func (s *Service) isGroupManaged(ctx context.Context) (bool, error) {
+func (s *Service) IsGroupManaged(ctx context.Context) (bool, error) {
```

Comment:
What is your thoughts on enforcing comments on public APIs?

Reply:
I generally like them :) sometimes I forget... Will add

Reply:
I was thinking about intro a PR to add to `make verify` to enforce public API comments. It might require some doc work to get us clean, but it would be nice to express that in our CI checks.

Reply:
I woudn't be opposed to that but it would require a lot of cleanup prior to being able to enforce since it would definitely fail with the codebase as-is. Do you mind opening an issue to track?